doc/audit/index.rst

```diff
@@ -53,28 +53,50 @@ Cleaning based on the config file:
 .. index:: retention time
 
 Using a config file you can define different retention times for the audit data.
-E.g. this way you can define, that audit entries about token listings can be deleted after one month,
+E.g. this way you can define, that audit entries about token listings can be deleted after
+one month,
 while the audit information about token creation will only deleted after ten years.
 
 The config file is a YAML format and looks like this:
```

Comment:
Hm, shouldn't we use `looks like this::` if a code snippet follows?

Reply:
I am always a bit confused by that!